.idea/AnnouncementSystem.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These xml files are IDE and compiler dependent, should be ignored from git because these files will be different for different IDE and compilers and OS'es.